### PR TITLE
Feat create sidenav and video listing page

### LINKED ISSUE #5: Create Sidenav component and video listing page
## To be created
- Sidenav
- Video card
- Video listing page

### PR DESCRIPTION
**Pull request by kjais1720**
PR for Issue #5 (closes issue #5 )
### Created
- Sidenav
- Video card
- Dropdown menu
- Video context
- setDocumentTitle utility
- Video Listing page
### Updated
- Categories db
- Videos db
## Preview
image

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Really Liked the use to truncate helper function to deal with long texts. I had used ellipsis but I liked this approach better. It gives us more control.
Also liked how you dealt with routing. 
Overall everything looks good and clean.

File: src/components/styles/sidenav.module.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.navigation{
+  position: sticky;
+  top: 6rem;
+  width: 75px;
```
Comment (rahulcs754) [inline src/components/styles/sidenav.module.css]: here i think you can use rem. if you don't use here. what was reason.

**Review by rahulcs754 — APPROVED** [review]
The code looks goods and clean always.
nice to  use function to pass function to fire a function.
Nice design

File: src/components/styles/sidenav.module.css (hunk 1 above)
Comment (kjais1720) [reply src/components/styles/sidenav.module.css]: Thanks fro pointing it out, I have fixed it in the latest commit

**Pull request merged**